lua/entities/gmod_wire_expression2/core/vector2.lua

```diff
@@ -1096,8 +1096,8 @@ __e2setcost(7)
 -- Returns a random vector4 between -1 and 1
 e2function vector4 randvec4()
 	local vec = { random()*2-1, random()*2-1, random()*2-1, random()*2-1 }
-	local length = sqrt( vec[1]^2+vec[2]^2+vec[3]^2+vec[4]^2 )
-	return { vec[1] / length, vec[2] / length }
+	local length = ( vec[1]^2+vec[2]^2+vec[3]^2+vec[4]^2 ) ^ 0.5 -- x ^ 0.5 <=> math.sqrt( x )
```

Comment:
I tested it inside gmod and sqrt was exactly the same as ^0.5, while math.sqrt is much much slower due to having to access the global scope to find the function. The reason this commit did this (changed sqrt to ^0.5) is because everywhere else in the file it was already done by someone else.